### PR TITLE
upgrade and set cache values

### PR DESCRIPTION
**Pull request by mcharriere**
- update app to 1.22.23
- force TCP for external zones as well

### CODE REVIEW

File: helm/k8s-dns-node-cache-app/templates/configmap.yaml [hunk 1]
```diff
@@ -52,10 +55,14 @@ data:
         template ANY AAAA {
                 rcode NOERROR
         }
-        cache {{ .Values.configmap.cache.success }}
+        cache {
+          prefetch 10 1m 20%
```
Comment (whites11) [inline helm/k8s-dns-node-cache-app/templates/configmap.yaml:59]: indentation wrong

File: helm/k8s-dns-node-cache-app/templates/configmap.yaml [hunk 2]
```diff
@@ -38,7 +39,9 @@ data:
     }
     ip6.arpa:{{ .Values.upstreamService.servicePort }} {
         errors
-        cache {{ .Values.configmap.cache.success }}
+        cache {
+          prefetch 10 1m 20%
```
Comment (whites11) [inline helm/k8s-dns-node-cache-app/templates/configmap.yaml:43]: indentation wrong

File: helm/k8s-dns-node-cache-app/templates/configmap.yaml [hunk 3]
```diff
@@ -13,8 +13,7 @@ data:
                 rcode NOERROR
         }
         cache {
-                success 9984 {{ .Values.configmap.cache.success }}
-                denial 9984 {{ .Values.configmap.cache.denial }}
+          prefetch 10 1m 20%
```
Comment (whites11) [inline helm/k8s-dns-node-cache-app/templates/configmap.yaml:16]: indentation wrong

**Comment by mcharriere** [comment]
tests in our clusters went well. a slight improvement in external records (now mostly fetched from cache) and no regressions

File: helm/k8s-dns-node-cache-app/templates/configmap.yaml [hunk 4]
```diff
@@ -13,8 +13,7 @@ data:
                 rcode NOERROR
         }
         cache {
-                success 9984 {{ .Values.configmap.cache.success }}
```
Comment (whites11) [inline helm/k8s-dns-node-cache-app/templates/configmap.yaml:16]: why did we remove these? afaik they come from upstream chart, there might be a reason why

Comment (mcharriere) [reply helm/k8s-dns-node-cache-app/templates/configmap.yaml:16]: After some digging I found this commit here explaing some issues for when `denial` is too high. I adjusted my PR to keep the 2 TTLs for each cache.

**Review by whites11 — APPROVED** [review]

**Review by njuettner — APPROVED** [review]

**Pull request merged**